product/README.md

```diff
@@ -59,35 +58,28 @@ For a full discussion of the situations where relative and absolute links are re
 ['Use of links'](../best-practices.md#use-of-links) section of the STAC best practices.
 
 ### Constraints Object
+
 Constraints are limitations imposed by the user and the data provider as to what data is desired, and what data is available. In addition to space, time, and product limitations, constraints filter the needs of the user and capacity of the provider through an API negotiation. Constraints build off STAC extensions and modify them in order to support ranges of values.
 
 Users of the Tasking API Should follow STAC Extension naming conventions when adding constraints.
 
+A Constraints Object is a definitions for fine-grained information, see the [JSON Schema Object](#json-schema-object) section for more.
 
-The constraints for a field can be specified in three ways:
+It is recommended to use [JSON Schema draft-07](https://json-schema.org/specification-links.html#draft-7)
+to align with the JSON Schemas provided by STAC. Empty schemas are not allowed.
 
-1. A set of all distinct values in an array: The set of values must contain at least one element and it is strongly recommended to list all values.
-2. A Range in a [Range Object](#range-object): Statistics by default only specify the range (minimum and maximum values),
-   but can optionally be accompanied by additional statistical values.
-   The range specified by the `minimum` and `maximum` properties can specify the potential range of values,
-   but it is recommended to be as precise as possible.
-3. Extensible JSON Schema definitions for fine-grained information, see the [JSON Schema Object](#json-schema-object)
-   section for more.
+For an introduction to JSON Schema, see "[Learn JSON Schema](https://json-schema.org/learn/)".
```

Comment:
I'm getting a 404 on this link. Is there a working URL?

Reply:
Probably an outdated URI. A PR would be appreciated.